docs/guides/apache-sites-enabled.md

```diff
@@ -8,14 +8,14 @@
 
 # Apache Web Server Multi-Site Setup
 
-Rocky Linux has many ways for you to set up a web site. This is just one method, using Apache, and is designed for use as a multi-site setup on a single server. While this method is designed for multi-site servers, it can also act as a base configuration for a single site server as well. 
+Rocky Linux has many ways for you to set up a website. One way is using Apache, which is for a multi-site setup on a single server. While this method is for multi-site servers, it can also act as a base configuration for a single-site webserver. 
 
-History fact: This server setup appears to have started with Debian-based systems, but it is perfectly adaptable to any Linux OS running Apache.
+Historical Fact: This server setup appears to have started with Debian-based systems, but it is perfectly adaptable to any Linux OS running Apache.
 
 ## Install Apache
-You'll likely need other packages for your web site. For instance, a version of PHP will almost certainly be required, and maybe a database or other package will be needed as well. Installing PHP along with httpd will get you the latest version of both from the Rocky Linux repositories. 
+You will likely need other packages for your website. PHP will almost certainly be required, and you will also need a database package. Installing PHP and httpd (Apache) will get you the latest version of both from the Rocky Linux repositories. 
 
```

Comment:
You don't necessarily need a database. Some sites don't require them.

Reply:
I have been working on the Internet from when it was GOPHER and Text-based Websites. Show me a website today that does not use a database; in fact why would you say that ‘PHP would almost certainly be required’ if there was no database?. 
The first paragraph is riddled with grammatical mistakes, show these two paragraphs to novices (and I have by the way) and will see which is more understandable.

Reply:
I've built websites, even in the last 5 years, without a database. You are correct that "most" do require them-but not all. Again, fix the grammatical mistakes. Don't change the meaning.